examples/prometheus/slo_exporter.yaml

```diff
@@ -0,0 +1,48 @@
+webServerListenAddress: "0.0.0.0:8080"
+afterPipelineShutdownDelay: "5s"
```

Comment:
We don't need this here.

In general, I'd say let's drop all configuration options which are not crucial for this example.

Reply:
Right, I thought that the default is quite high but I checked and it's zero so yes, no need for it here :+1: